[2023-12-05 21:56:48,847: INFO: main: Welcome to our custom log]
[2023-12-06 14:17:06,659: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-12-06 14:17:06,659: ERROR: main: [Errno 2] No such file or directory: '/workspaces/newsite/IOT-Network-Intrusion-Detection-System/src/IOT_NIDS/config/config.yaml']
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    data_ingestion.main()
  File "/workspaces/newsite/IOT-Network-Intrusion-Detection-System/src/IOT_NIDS/pipeline/stage_01_data_ingestion.py", line 13, in main
    config = ConfigurationManager()
  File "/workspaces/newsite/IOT-Network-Intrusion-Detection-System/src/IOT_NIDS/config/configuration.py", line 12, in __init__
    self.config = read_yaml(Path(__file__).resolve().parent / "config.yaml")
  File "/home/codespace/.conda/envs/nids/lib/python3.8/site-packages/ensure/main.py", line 849, in __call__
    return_val = self.f(*args, **kwargs)
  File "/workspaces/newsite/IOT-Network-Intrusion-Detection-System/src/IOT_NIDS/utils/common.py", line 38, in read_yaml
    raise e
  File "/workspaces/newsite/IOT-Network-Intrusion-Detection-System/src/IOT_NIDS/utils/common.py", line 31, in read_yaml
    with open(path_to_yaml) as yaml_file:
FileNotFoundError: [Errno 2] No such file or directory: '/workspaces/newsite/IOT-Network-Intrusion-Detection-System/src/IOT_NIDS/config/config.yaml'
[2023-12-06 14:21:15,880: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-12-06 14:21:15,882: INFO: common: yaml file: /workspaces/newsite/IOT-Network-Intrusion-Detection-System/src/IOT_NIDS/config/config.yaml loaded successfully]
[2023-12-06 14:21:15,882: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-06 14:21:15,882: INFO: common: created directory at: artifacts]
[2023-12-06 14:21:15,882: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-06 14:21:15,883: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1OEAJcEe1ycMRrtUpAMAYTBb-8txf_1Er/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2023-12-06 14:21:18,819: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1OEAJcEe1ycMRrtUpAMAYTBb-8txf_1Er/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2023-12-06 14:21:19,687: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-12-06 23:09:19,191: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:09:19,195: INFO: common: created directory at: artifacts]
[2023-12-06 23:09:54,598: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:09:54,600: INFO: common: created directory at: artifacts]
[2023-12-06 23:09:54,601: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:15:34,004: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:15:34,006: INFO: common: created directory at: artifacts]
[2023-12-06 23:15:34,006: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:17:51,380: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:17:51,381: INFO: common: created directory at: artifacts]
[2023-12-06 23:17:51,382: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:20:27,540: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:20:27,542: INFO: common: created directory at: artifacts]
[2023-12-06 23:20:27,543: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:22:19,442: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:22:19,443: INFO: common: created directory at: artifacts]
[2023-12-06 23:22:19,444: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:29:54,629: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:29:54,630: INFO: common: created directory at: artifacts]
[2023-12-06 23:29:54,631: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:31:30,212: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:31:30,213: INFO: common: created directory at: artifacts]
[2023-12-06 23:31:30,214: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:34:40,818: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:34:40,819: INFO: common: created directory at: artifacts]
[2023-12-06 23:34:40,820: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:35:56,163: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:35:56,164: INFO: common: created directory at: artifacts]
[2023-12-06 23:35:56,165: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:38:02,298: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:38:02,299: INFO: common: created directory at: artifacts]
[2023-12-06 23:38:02,299: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:44:29,208: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:44:29,210: INFO: common: created directory at: artifacts]
[2023-12-06 23:44:29,211: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:46:30,317: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=4, redirect=5, status=5)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='dagshub.com', port=443): Read timed out. (read timeout=120)")': /RCgit123/IOT-Network-Intrusion-Detection-System.mlflow/api/2.0/mlflow/runs/create]
[2023-12-06 23:47:23,524: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2023-12-06 23:56:55,001: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-06 23:56:55,002: INFO: common: created directory at: artifacts]
[2023-12-06 23:56:55,002: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-06 23:58:55,893: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=4, redirect=5, status=5)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='dagshub.com', port=443): Read timed out. (read timeout=120)")': /RCgit123/IOT-Network-Intrusion-Detection-System.mlflow/api/2.0/mlflow/runs/create]
[2023-12-07 00:00:35,702: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-07 00:00:35,703: INFO: common: created directory at: artifacts]
[2023-12-07 00:00:35,704: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-07 00:01:53,815: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-07 00:01:53,816: INFO: common: created directory at: artifacts]
[2023-12-07 00:01:53,817: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-07 00:01:56,081: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2023-12-07 00:17:06,146: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-07 00:17:06,147: INFO: common: created directory at: artifacts]
[2023-12-07 00:17:06,148: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-07 00:19:07,083: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=4, redirect=5, status=5)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='dagshub.com', port=443): Read timed out. (read timeout=120)")': /RCgit123/IOT-Network-Intrusion-Detection-System.mlflow/api/2.0/mlflow/runs/create]
[2023-12-07 00:19:08,668: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2023-12-07 00:29:26,317: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-07 00:29:26,318: INFO: common: created directory at: artifacts]
[2023-12-07 00:29:26,318: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-07 00:31:27,313: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=4, redirect=5, status=5)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='dagshub.com', port=443): Read timed out. (read timeout=120)")': /RCgit123/IOT-Network-Intrusion-Detection-System.mlflow/api/2.0/mlflow/runs/create]
[2023-12-07 00:32:21,948: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2023-12-07 05:56:15,185: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-07 05:56:15,189: INFO: common: created directory at: artifacts]
[2023-12-07 05:56:15,190: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-07 05:57:12,506: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2023-12-07 06:08:18,040: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-07 06:08:18,041: INFO: common: created directory at: artifacts]
[2023-12-07 06:08:18,042: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-07 06:10:19,050: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=4, redirect=5, status=5)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='dagshub.com', port=443): Read timed out. (read timeout=120)")': /RCgit123/IOT-Network-Intrusion-Detection-System.mlflow/api/2.0/mlflow/runs/create]
[2023-12-07 06:11:12,848: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2023-12-07 06:14:46,718: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-07 06:14:46,719: INFO: common: created directory at: artifacts]
[2023-12-07 06:14:46,720: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-07 06:15:40,899: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2023-12-07 06:18:25,805: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-07 06:18:25,807: INFO: common: created directory at: artifacts]
[2023-12-07 06:18:25,808: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-07 06:18:28,068: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2023-12-07 06:21:04,362: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-07 06:21:04,363: INFO: common: created directory at: artifacts]
[2023-12-07 06:21:04,364: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-07 06:21:06,754: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2023-12-07 06:22:04,592: INFO: common: yaml file: config/config.yaml loaded successfully]
[2023-12-07 06:22:04,594: INFO: common: created directory at: artifacts]
[2023-12-07 06:22:04,594: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-07 06:22:59,074: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2023-12-07 06:53:54,825: INFO: main: >>>>>> stage Model Evaluation stage started <<<<<<]
[2023-12-07 06:53:54,829: INFO: common: yaml file: /workspaces/newsite/IOT-Network-Intrusion-Detection-System/src/IOT_NIDS/config/config.yaml loaded successfully]
[2023-12-07 06:53:54,829: INFO: common: created directory at: artifacts]
[2023-12-07 06:53:54,829: INFO: common: created directory at: artifacts/model_evaluation]
[2023-12-07 06:54:48,434: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2023-12-07 06:54:52,099: INFO: main: >>>>>> stage Model Evaluation stage completed <<<<<<

x==========x]
